index.html

```diff
@@ -8,7 +8,7 @@
     <h2><b>BSidesSF</b></h2>
 
     <p>
-      BSidesSF 2020 Call for participation is <b>now open!</b> If you have a talk or workshop that you would like to bring to the community please submit an entry to our <a hfef="/cfp.html">Call for Participation</a>.<br/>
+      BSidesSF 2020 Call for participation is <b>now open!</b> If you have a talk or workshop that you would like to bring to the community please submit an entry to our <a hfef="/cfp.html">Call For Participation</a>.<br/>
```

Comment:
Please use exactly what I provided. There are numerous issues right now. :-)

Reply:
The comment showed up as quote formatting rather than code formattting, am I interpreting correctly the changes you wanted are:

- Capitol F in Call For Participation
- My stupid inability to type `href`
- Comma after community
?

Reply:
- And capitol P. Wow do I have typos! 

Thanks for the typo checking, @reedloden I appreciate it.